keeper-gradle-plugin/src/test/kotlin/com/slack/keeper/KeeperFunctionalTest.kt

```diff
@@ -262,40 +269,23 @@ private fun String.prefixIfNot(prefix: String) =
     if (this.startsWith(prefix)) this else "$prefix$this"
 
 @Language("PROGUARD")
-private val EXPECTED_PRINT_RULES_RULES = """
-  -keep class com.slack.keeper.sample.TestOnlyClass {
-    public static void testOnlyMethod();
-  }
-  -keep class com.slack.keeper.sample.TestOnlyKotlinClass {
-    public void testOnlyMethod();
-    com.slack.keeper.sample.TestOnlyKotlinClass INSTANCE;
-  }
-  -keeppackagenames
```

Comment:
Why is this one not expected in tracereferences?

Reply:
Not an issue now that I restored the original version in b68f65a